upnp / dlna

Hi everyone and first a huge thank you for porting Jellyfin under yunohost (special thanks @tituspijean )
I come to you to solve my problem, let me explain.
I am trying to use Jellyfin to play my music on my upnp / dlna Harman Kardon audio amplifier.
First remark I am systematically obliged to retype the command "sudo yunohost firewall allow UDP 1900" each time my server starts up, otherwise jellyfin and my amplifier cannot communicate ..
So after typing this command, my amp sees the Jellyfin server and conversely Jellyfin sees my amp well .. So I can play my music from Jellyfin-web cast on my amp ..
This is where my real problem comes in, at each end of the song I have the beginning of the song which starts over again before finally moving on to the next song, very strange but above all very unpleasant.
 I noticed that the time progression of the songs became Naan: Naan at the end of the song, and I also have some titles which are directly played without a time value which blocks the playback ..
If anyone has an idea to solve this I am interested ..
Hope the solution exists, thanks again ..

Not sure if it's related, but I'm currently having the diagnosis report send me this email every single day:

```
The automatic diagnosis on your YunoHost server identified some issues on your server. You will find a description of the issues below. You can manage those issues in the 'Diagnosis' section in your webadmin.

---

=================================
Ports exposure (ports)
=================================

[ERROR] Port 1900 is not reachable from outside.
  - Exposing this port is needed for [?] features (service jellyfin)
  - To fix this issue, you most probably need to configure port forwarding on your internet router as described in https://yunohost.org/isp_box_config

[ERROR] Port 7359 is not reachable from outside.
  - Exposing this port is needed for [?] features (service jellyfin)
  - To fix this issue, you most probably need to configure port forwarding on your internet router as described in https://yunohost.org/isp_box_config
```

I'm currently using Wireguard to have my personal server access the Internet outside of my network (my ISP does not allow incoming traffic straight to my modem, like my old one did). Do I need to do some special configuration on my Wireguard server? I've already tried routing those two ports directly to my device, but I'm not sure if that's causing more problems instead.

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/in
 Jellyfin enables you to collect, manage, and stream your media. Run the Jellyfin server on your system and gain access to the leading free-software entertainment system, bells and whistles included.
 
 
-**Shipped version:** 10.8.10~ynh1
+**Shipped version:** 10.8.10~ynh2
 
 **Demo:** https://demo.jellyfin.org/stable/web/index.html
 

--- a/README_fr.md
+++ b/README_fr.md
@@ -19,7 +19,7 @@ Si vous n’avez pas YunoHost, regardez [ici](https://yunohost.org/#/install) po
 Jellyfin vous permet de collecter, gérer et diffuser vos médias. Exécutez le serveur Jellyfin sur votre système et accédez au principal système de divertissement à logiciel libre.
 
 
-**Version incluse :** 10.8.10~ynh1
+**Version incluse :** 10.8.10~ynh2
 
 **Démo :** https://demo.jellyfin.org/stable/web/index.html
 

--- a/conf/LDAP-Auth.xml
+++ b/conf/LDAP-Auth.xml
@@ -1,16 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PluginConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <LdapServer>127.0.0.1</LdapServer>
-  <LdapBaseDn>ou=users,dc=yunohost,dc=org</LdapBaseDn>
   <LdapPort>389</LdapPort>
-  <LdapSearchAttributes>uid, mail</LdapSearchAttributes>
-  <LdapUsernameAttribute>uid</LdapUsernameAttribute>
-  <LdapSearchFilter>(permission=cn=jellyfin.main,ou=permission,dc=yunohost,dc=org)</LdapSearchFilter>
-  <LdapAdminFilter>(permission=cn=jellyfin.admin,ou=permission,dc=yunohost,dc=org)</LdapAdminFilter>
-  <LdapBindUser></LdapBindUser>
-  <LdapBindPassword></LdapBindPassword>
-  <CreateUsersFromLdap>true</CreateUsersFromLdap>
   <UseSsl>false</UseSsl>
   <UseStartTls>false</UseStartTls>
   <SkipSslVerify>false</SkipSslVerify>
+  <LdapBindUser />
+  <LdapBindPassword />
+  <LdapBaseDn>ou=users,dc=yunohost,dc=org</LdapBaseDn>
+  <LdapSearchFilter>(permission=cn=jellyfin.main,ou=permission,dc=yunohost,dc=org)</LdapSearchFilter>
+  <LdapAdminBaseDn />
+  <LdapAdminFilter>(permission=cn=jellyfin.admin,ou=permission,dc=yunohost,dc=org)</LdapAdminFilter>
+  <LdapSearchAttributes>uid, mail</LdapSearchAttributes>
+  <EnableCaseInsensitiveUsername>false</EnableCaseInsensitiveUsername>
+  <CreateUsersFromLdap>true</CreateUsersFromLdap>
+  <AllowPassChange>false</AllowPassChange>
+  <LdapUsernameAttribute>uid</LdapUsernameAttribute>
+  <LdapPasswordAttribute>userPassword</LdapPasswordAttribute>
+  <EnableAllFolders>true</EnableAllFolders>
+  <EnabledFolders>
+    <string>db36da0754ef18efa77eabef2d57a80d</string>
+    <string>db4c1708cbb5dd1676284a40f2950aba</string>
+  </EnabledFolders>
+  <PasswordResetUrl />
 </PluginConfiguration>

--- a/conf/nginx.conf
+++ b/conf/nginx.conf
@@ -1,12 +1,13 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
 location __PATH__/ {
+  
   proxy_pass http://127.0.0.1:__PORT____PATH__/;
   proxy_pass_request_headers on;
   proxy_set_header Host $host;
   proxy_set_header X-Real-IP $remote_addr;
   proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
   proxy_set_header X-Forwarded-Proto $scheme;
-  proxy_set_header X-Forwarded-Host $http_host;
+  proxy_set_header X-Forwarded-Host $host;
   proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection $http_connection;
 

--- a/manifest.json
+++ b/manifest.json
@@ -6,7 +6,7 @@
         "en": "Media System that manage and stream your media",
         "fr": "Système multimédia qui gère et diffuse vos médias"
     },
-    "version": "10.8.10~ynh1",
+    "version": "10.8.10~ynh2",
     "url": "https://jellyfin.org",
     "upstream": {
         "license": "GPL-2.0-only",

--- a/scripts/install
+++ b/scripts/install
@@ -191,13 +191,7 @@ ynh_use_logrotate
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
 
-# Build the ports list
-needed_ports=()
-(( $discovery_service == 1 )) && needed_ports+=( "$discovery_service_port" )
-(( $discovery_client == 1 )) && needed_ports+=( "$discovery_client_port" )
-
-# Integrate service and require to expose the ports if needed
-yunohost service add $app --description="Jellyfin media center" ${needed_ports:+--needs_exposed_ports} "${needed_ports[@]}"
+yunohost service add $app --description="Jellyfin media center"
 
 #=================================================
 # START SYSTEMD SERVICE

--- a/scripts/restore
+++ b/scripts/restore
@@ -164,13 +164,7 @@ ynh_restore_file --origin_path="/etc/logrotate.d/$app"
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
 
-# Build the ports list
-needed_ports=()
-(( $discovery_service == 1 )) && needed_ports+=( "$discovery_service_port" )
-(( $discovery_client == 1 )) && needed_ports+=( "$discovery_client_port" )
-
-# Integrate service and require to expose the ports if needed
-yunohost service add $app --description="Jellyfin media center" ${needed_ports:+--needs_exposed_ports} "${needed_ports[@]}"
+yunohost service add $app --description="Jellyfin media center"
 
 #=================================================
 # START SYSTEMD SERVICE

--- a/scripts/upgrade
+++ b/scripts/upgrade
@@ -215,13 +215,7 @@ ynh_use_logrotate --non-append
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
 
-# Build the ports list
-needed_ports=()
-(( $discovery_service == 1 )) && needed_ports+=( "$discovery_service_port" )
-(( $discovery_client == 1 )) && needed_ports+=( "$discovery_client_port" )
-
-# Integrate service and require to expose the ports if needed
-yunohost service add $app --description="Jellyfin media center" ${needed_ports:+--needs_exposed_ports} "${needed_ports[@]}"
+yunohost service add $app --description="Jellyfin media center"
 
 #=================================================
 # START SYSTEMD SERVICE

